serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,257 @@
+----
+
+Once started, you can request the provided endpoint:
+
+[source,shell]
+----
+curl -X POST -H 'Content-Type:application/json' http://localhost:8080/hello_world
+----
+
+The response should be similar to:
+
+[source,shell]
+----
+{"id":"efb59bfa-ad9c-4062-a6d2-2d9184dd4b3d","workflowdata":{"greeting":"Hello World","mantra":"Serverless Workflow is awesome!"}}
+----
+
+Hit `CTRL+C` to stop the application, or keep it running and enjoy the blazing fast hot-reload.
+
+== Testing
+
+Now, let's add a test to make sure everything works as expected. But first, remove the
+auto-generated tests `org.kie.kogito.examples.GreetingResourceTest` and
+`org.kie.kogito.examples.NativeGreetingResourceIT`.
+
+Then, create the `org.kie.kogito.examples.HelloWorldTest` test class with the following content:
+
+[source,java]
+----
+package org.kie.kogito.examples;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+class HelloWorldTest {
+
+    @BeforeAll
+    static void init() {
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
+    }
+
+    @Test
+    void testHelloEndpoint() {
+        given()
+                .contentType(ContentType.JSON)
+                .accept(ContentType.JSON)
+                .when().post("/hello_world")
+                .then()
+                .statusCode(201)
+                .body("workflowdata.greeting", is("Hello World"))
+                .body("workflowdata.mantra", is("Serverless Workflow is awesome!"));
+    }
+}
+----
+
+Now you're ready to run the test:
+
+[source,shell]
+----
+mvn clean verify
+----
+
+You can also run the test from your IDE directly (be sure you stopped the application first).
+
+If you want to know more about testing Quarkus applications, check out the https://quarkus.io/guides/getting-started-testing[Quarkus guide].
+
+== What's next?
+
+This guide covered the creation of a Serverless Workflow application using Kogito.
+However, there is much more. We recommend continuing the journey with the getting started guides
+link:cncf-serverless-workflow-specification-support.html[CNCF Serverless Workflow Specification Support]
+and
+link:getting-familiar-with-our-tooling.html[Getting Familiar With Our Tooling].
+Also, you can go to the Cloud category from here to understand how to deploy your application in the cloud.
```

Comment:
"Cloud category" should be a link

Reply:
"Cloud" category is not a page. So, it doesn't have a link.

To understand what I'm talking about, see this example from the Antora docs:
[This](https://docs.antora.org/antora/latest/navigation/files-and-lists/) is a category and a page. So, it has a link. But the category above that (Navigation) is only a category, which doesn't have a link.

We can turn "Cloud" into a page and add links to the guides under this category to this page. But we'll end up with two places to update when adding a new guide (this page and the navigation file). This might not be a good idea.
Any thoughts on this @ricardozanini and @akumar074?

Reply:
Oh, right. Once we have a website we might have a link like Quarkus guides have: "https://quarkus.io/guides/#cloud"